HomepageV2.css

```diff
@@ -0,0 +1,335 @@
+html {
+    scroll-behavior: smooth;
+}
+body {
+    -ms-overflow-style: none;  // IE 10+
+    scrollbar-width: none;  // Firefox
+
+}
+body::-webkit-scrollbar { 
+    display: none;  // Safari and Chrome
+}
+
+@font-face {
+    font-family: "neutra";
+    src: url("NeutraText-book.otf") format("opentype");
+}
+
+@font-face {
+    font-family: "neutra-bold";
+    src: url("NeutraText-bold.otf") format("opentype");
+}
+
+@font-face {
+    font-family: "robot";
+    src: url("Robot Renegades.otf") format("opentype");
+}
+
+body {
+    padding: 0;
+    margin: 0;    
+    font-family: "neutra-bold";
+}
+
+.teamLogo {
+    color: rgb(249, 211, 139);
+    position: relative;
+    top: -31px;
+    border: 2px solid rgb(249, 211, 139);
+    padding: 6px;
+    display: inline;
+    padding-top: 2px;
+    padding-bottom: 2px;
+    font-size: 15px;
+    z-index: 200;
+}
+
+.contactText1 {
```

Comment:
Name this something more descriptive or add comments to help other people understand?